substrate/client/rpc-spec-v2/src/chain_head/chain_head_follow.rs

```diff
@@ -202,13 +212,40 @@ where
 
 				for pair in blocks.zip(parents) {
```

Comment:
Unrelated to this PR but I don't understand the logic with `zip` here.

Parents = blocks.len() + 1

Then the `zip` will ignore the last item in blocks is that indented?